java/src/jmri/web/servlet/panel/ControlPanelServlet.java

```diff
@@ -67,7 +67,11 @@ protected String getXmlPanel(String name) {
                         Element e = ConfigXmlManager.elementFromObject(sub);
                         if (e != null) {
                             if ("signalmasticon".equals(e.getName())) {  //insert icon details into signalmast
-                                e.addContent(getSignalMastIconsElement(e.getAttributeValue("signalmast")));
+                                String imageset = e.getAttributeValue("imageset") ;
```

Comment:
I suggest you move this null check into the method getSignalMastIconsElement(String, String) itself. That way you are not duplicating that check before every call to that method.